examples/aws-cloudwatch/capacity_planning/ec2.juttle

```diff
@@ -22,7 +22,7 @@ const instance_prices = {
     't2.medium': 0.05,
 };
 
-read mysql -table 'aws_aggregation' -from :0: -to :end:
+read mysql -table 'aws_aggregation' -last :1h:
```

Comment:
Yeah, I was making it consistent with the other capacity planning programs that all use -last :1h:.